npm-debug.log

```diff
@@ -0,0 +1,23 @@
+0 info it worked if it ends with ok
```

Comment:
I'm not sure how this even got in here.  I think I tried to run an npm command forgetting this was a rails app.  I'll add this to the git ignore.